### ISSUE TITLE
Ring of Elements animation remains

### ISSUE THREAD

**Issue by keywok** [ArtimosPenguidor/animation-transmog#20]
When selecting a different teleport animation the ring of elements animation plays over the top still and isn't hidden or replaced like the default teleport animation.

![RuneLite_21-11-24_19-22-46](https://github.com/user-attachments/assets/c3912b15-6344-4fe6-bc2b-d64f4311dee3)

**Comment by BarelyFunctionalCode** [comment]
Yeah unfortunately there is no simple way to cover every teleport, so for every unique teleport animation in the game I have to add it to a list for the plugin to overwrite it. 

I'll add this to the list on the next update. Thanks!
